### DISCUSSION TITLE
Change the validation message on nested properties

### DISCUSSION THREAD

**Discussion by edalzell** [spatie/laravel-data#734] [Q&A]
In v3, I have an object with a nested property: 
![CleanShot 2024-04-03 at 14 03 01@2x](https://github.com/spatie/laravel-data/assets/6069653/4bb08b4b-1486-4c2c-8626-0c5ea682a103)

In that data class I have: 
![CleanShot 2024-04-03 at 14 03 33@2x](https://github.com/spatie/laravel-data/assets/6069653/44068ba3-d8a8-493a-8d6f-b20be2e7618c)

When I validate the object I get this error: 
![CleanShot 2024-04-03 at 14 03 46@2x](https://github.com/spatie/laravel-data/assets/6069653/6fe0b01b-f7a3-4ef2-840c-94de5fe1f16d)

Is there a nice way to customize that message?

**Comment by rubenvanassche** [comment]
This? https://spatie.be/docs/laravel-data/v4/validation/working-with-the-validator#content-overwriting-messages

**Reply by edalzell** [reply]
I’m still using v3 for the moment

**Reply by rubenvanassche** [reply]
v4 didn't change anything about validation so should also be present in v3. Though the docs were rewritten so probably you'll find it in another place.
